tests/operator/operator.go

```diff
@@ -3440,3 +3386,51 @@ func eventuallyDeploymentNotFound(name string) {
 		return err
 	}).WithTimeout(15 * time.Second).WithPolling(1 * time.Second).Should(MatchError(errors.IsNotFound, "not found error"))
 }
+
+func allPodsAreReady(kv *v1.KubeVirt) {
```

Comment:
I think it'd be a bit clearer this way
```suggestion
func allKvInfraPodsAreReady(kv *v1.KubeVirt) {
```